### DISCUSSION TITLE
Remove sound stacking volume increase

### DISCUSSION THREAD

**Discussion by WeldRS** [runelite/runelite#15352] [Ideas]
Would like to see OSRS sounds (particularly combat sounds) that 'stack' or overlap and increase in volume when multiple hits are occuring at the same time removed or taken down to the same volume level as any other sounds.

Some examples I can think of

-Barrage/burst spells on multiple targets
-Chinchompa hits on multiple targets
-Multiple people attacking the same monster at the same time
-Multiple people teleporting near you 

These are just a few, there are many different instances where the volume of OSRS sounds even at max volume increases a ridiculous amount when stacked and I find myself constantly playing with the in game volume slider depending on what I'm doing.

It would be nice to have all sounds at the same level or just remove the sound stack volume increase from happening.

**Comment by xxEzri** [comment]
Not sure if it's reasonably doable but if possible i think the best solution would be to delay the subsequent plays of the same sound slightly - this is what we would do with full engine control. The root cause is that all of the sounds are being fired off exactly simultaneously on the game tick, so they stack in a way that amplifies the volume to a ridiculous level and also prevents you from distinguishing between 1 or 2 or 8 plays of the same sound. Offsetting fixes both.

Here's a quick audio sample i threw together which has the following: 1 ice barrage sound, then 8 stacked together (what we have right now) then 8 with each sound delayed a little bit compared to the one before it. You can hear that it's distinguishable as ice barrage hitting a bunch of things rather than just one thing and that it's also not crazy loud like the stacked sound. Parameters like the amount of delay and how the delay is applied (if there's a linear offset between each sound or a random start time etc) can also be tweaked if the start time can be controlled.

https://cdn.discordapp.com/attachments/886739055994343495/997813941893857340/Ice_Barrage_Single_8stack_8spread.wav
